README.md

```diff
@@ -221,6 +223,44 @@ const burn = await client.getBurn(newBurnUid);
 const burns = await client.getBurnsByUserUid(newUserUid);
 ```
 
+### Allocate
+
+The allocate methods exposed by the sdk are used to allocate funds in a reward contract, to an app user
+
+```typescript
```

Comment:
Some disconnect here between the endpoint being called 'allocate', but the proper grammar being 'allocation'
wdyt? @don1989